data/FilterList.json

```diff
@@ -9745,191 +9745,191 @@
   },
   {
     "id": 481,
-    "chatUrl": null,
-    "description": null,
-    "descriptionSourceUrl": null,
-    "discontinuedDate": null,
-    "donateUrl": null,
-    "emailAddress": "mail.energized@protonmail.com",
-    "forumUrl": null,
-    "homeUrl": "https://ador.chorompotro.com/energized",
-    "issuesUrl": "https://github.com/EnergizedProtection/EnergizedHosts/issues",
-    "licenseId": 2,
-    "name": "Energized Ad Protection",
+    "chatUrl": "https://t.me/EnergizedProtection",
+    "description": "True Lite Hosts from Energized. Only 1MB in Size.",
+    "descriptionSourceUrl": "https://github.com/EnergizedProtection/block",
+    "discontinuedDate": null,
+    "donateUrl": "shadmansaleh@codeian.com",
```

Comment:
oops, this one still doesn't look right. I think you updated `emailAddress` rather than `donateUrl`.